src/pages/Homepage/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from "react";
+import CategoryCard from "./components/CategoryCard";
+import HeroSection from "./components/HeroSection";
+import Banner from "images/banner.png";
+import axios from "axios";
+import "./index.less";
+
+const Homepage = () => {
+  const [categories, setCategories] = useState([]);
+
+  const getCategories = async () => {
+    try {
+      const { data } = await axios.get("/api/categories");
+      setCategories(data.categories);
+    } catch (error) {
+      console.log(error);
+    }
+  };
```

Comment:
Suggestion: you may make a `utils` folder to have all network related requests like getProducts, getCategories, login etc..

Reply:
Sure would update it. Thanks